nncf/experimental/common/tensor_statistics/statistics.py

```diff
@@ -252,6 +314,17 @@ def __eq__(self, other: Any) -> bool:
         )
         return mean_values_equal
 
+    def get_data_to_dump(self):
+        tensor_method = get_tensor_method(self.mean_values[0].backend)
+        return {
+            self.MEAN_STAT: tensor_method([mean_value.data for mean_value in self.mean_values]),
+            self.SHAPE_STAT: tensor_method([[dim.data for dim in shape] for shape in self.shape_values]),
+        }
+
+    def load_data(self, loaded_data: Dict[str, TTensor]) -> None:
+        self.shape_values = [tuple(shape.tolist()) for shape in loaded_data[self.SHAPE_STAT]]
```

Comment:
Direct call .tolist() method for any tensor type
Looks like it should be done like `Tensor(loaded_data[self.SHAPE_STAT]).tolist()`

Reply:
updated. Please check